app/src/main/java/com/vitorpamplona/amethyst/ui/navigation/DrawerContent.kt

```diff
@@ -135,12 +133,10 @@ fun ProfileContent(baseAccountUser: User, modifier: Modifier = Modifier, scaffol
         }
 
         Column(modifier = modifier) {
-            AsyncImageProxy(
+            RobohashAsyncImageProxy(
+                robot = accountUser.pubkeyHex,
                 model = ResizeImage(accountUser.profilePicture(), 100.dp),
                 contentDescription = stringResource(id = R.string.profile_image),
-                placeholder = BitmapPainter(RoboHashCache.get(ctx, accountUser.pubkeyHex)),
-                fallback = BitmapPainter(RoboHashCache.get(ctx, accountUser.pubkeyHex)),
-                error = BitmapPainter(RoboHashCache.get(ctx, accountUser.pubkeyHex)),
```

Comment:
I removed placeholder/fallback/error parameters on all calls to the image composables. I'm not sure if they should still be provided, maybe with some static image like the Amethyst logo, just in case.